<launch>
    <arg name="image_topics" default="[/usb_cam/image_raw]"/>
    <node name="image_decoder" pkg="image_compressor" type="image_uncompressor_main" output="screen">
        <rosparam param="uncompress_video_topics" subst_value="True">$(arg image_topics)</rosparam>
    </node>
</launch>